src/utils.js

```diff
@@ -29,3 +29,15 @@ export function isWindowOrWorker() {
     )
   );
 }
+
+export function isBrowserEnv() {
+  return typeof window !== 'undefined';
+}
+
+export function createBrowserSafeString(toBeConverted) {
+  const convertedString = toBeConverted.toString('base64')
```

Comment:
No not really since its base64 already. I can change it to skip that for browser strings if we want, but I figured since it doesn't really do anything then the extra 'if' was kinda unnecessary. I can change it if you want though

Reply:
👍🏼